File: Interviews/GFG/Daily-Problems/2022/October/Day-24/Earthquake-and-the-Paint-Shop.py
class alphanumeric:
    def __init__ (self, name, count):
        self.name = name
        self.count = count
class Solution:
    def sortedStrings(self, N, A):
        mp, ans = {}, []
        for s in A:
            mp[s] = mp.get(s, 0) + 1
        for key in sorted(mp.keys()):
            ans.append(alphanumeric(key, mp[key]))
        
        return ans

File: Interviews/GFG/random-problems/Easy/Earthquake-and-the-Paint-Shop/Earthquake-and-the-Paint-Shop.cpp
struct alphanumeric {
    string name;
    int count;
};
class Solution {
public:
    vector<alphanumeric> sortedStrings(int N, vector<string> A) {
        map<string, int> mp;
        for (string s : A) {
            mp[s] += 1;
        }

        vector<alphanumeric> ans;
        for (auto i : mp) {
            ans.push_back({i.first, i.second});
        }

        return ans;
    }
};